metrics-graphite/src/main/java/com/codahale/metrics/graphite/MetricTypes.java

```diff
@@ -0,0 +1,23 @@
+package com.codahale.metrics.graphite;
+
+public class MetricTypes {
```

Comment:
I think we could make it an enum and move it to `metrics-core`. Upsides:

* Additional type-safety;
* We can use an `EnumSet` for storing a blacklist of metric types;
* Other reporters could access it if needed.